css/main.css

```diff
@@ -1,32 +1,56 @@
-@import "lato.css";
+@import "../fonts/inter/inter.css";
 
 :root {
   --main-breakpoint: 60em;
 
   /* Serif fonts for body text. Easier to read for larger chunks */
-  --serif-fonts: Lato, Cambria, Garamon, Palatino, P052, serif;
+  --serif-fonts: "Inter", var(--bs-font-serif);
   /* Sans-serif for smoother fonts. Useful for titles and shorter chunks */
-  --sans-serif-fonts: Lato, sans-serif;
+  --sans-serif-fonts: "Inter", var(--bs-font-sans-serif);
```

Comment:
The old font looked a bit cheap to me, will be great if this new one feels better :)